src/org/benf/cfr/reader/bytecode/analysis/parse/lvalue/FieldVariable.java

```diff
@@ -133,7 +133,7 @@ public Dumper dumpInner(Dumper d) {
                 object.dumpWithOuterPrecedence(d, getPrecedence(), Troolean.NEITHER).separator(".");
             }
         }
-        return d.fieldName(getFieldName(), getField(), getOwningClassType(), isHiddenDeclaration(), false);
+        return d.fieldName(getFieldName(), getDescriptor(), getOwningClassType(), isHiddenDeclaration(), false,false);
```

Comment:
```suggestion
        return d.fieldName(getFieldName(), getDescriptor(), getOwningClassType(), isHiddenDeclaration(), false, false);
```

Reply:
Sorry, missed this nice patch :weary: was out of home all day

Reply:
No dont worry, was my bad for merging it.

Should possibly look at getting checkstyle going.